examples/webserver/threadpool/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
This file is not needed, please do not include it.

Reply:
Fixed.